index.html

```diff
@@ -0,0 +1,397 @@
+                      </div>
+                    </div>
+                  </li>
+                </ul>
+
+                <div class="framework">
+                  <h3 class="Aboutme-subtitle">Frameworks</h3>
+                  <div class="framework-spacer"></div>
+                  <div class="framework-item">
+                    <div class="Aboutme-image">
+                      <img
+                        src="./img/arrow-forward.png"
+                        alt=""
+                        width="10"
+                        height="10"
+                      />
+                    </div>
+                  </div>
+                </div>
+                <hr class="Abt-border" />
+                <div class="framework">
+                  <h3 class="Aboutme-subtitle">Skills</h3>
+                  <div class="framework-spacer1"></div>
+                  <div class="framework-item">
+                    <div class="Aboutme-image5">
+                      <img
+                        src="./img/arrow-forward.png"
+                        alt=""
+                        width="10"
+                        height="10"
+                      />
+                    </div>
+                  </div>
+                </div>
+                <hr class="Aboutme-border" />
+              </div>
+            </div>
+          </section>
+
+          <section >
+            <div class="contact-container">
+              <h3 class="intro-text">Contact Me</h3>
+              <p>
+                If you have an application you are interested in developing, a
+                feature that you need built or a project that needs coding. I’d
+                love to help with it
+              </p>
+            <form action="https://formspree.io/f/mgedddej" method="post">
+                <ul>
+                  <li>
+                    <input
+                      name="Name"
+                      id="text"
+                      type="text"
+                      placeholder="Your name"
+                      maxlength="30"
+                      required
+                    />
+                  </li>
+                  <li>
+                    <input
+                      name="Email"
+                      id="email"
+                      type="email"
+                      placeholder="Your email address"
+                      required
+                    />
+                  </li>
+                  <li>
+                    <textarea
+                      name="comment"
+                      placeholder="Write your message here"
+                      maxlength="500"
+                      required
+                    ></textarea>
+                  </li>
+                  <li>
+                    <button type="submit" value="Submit" class="btn">
+                      Get In Touch
+                    </button>
+                  </li>
+                </ul>
+              </form>
+            </div>,
+          </section>
```

Comment:
Kindly, stick to the design provided as much as possible. This section has a background image please add the image as background inside your CSS file. And also it has a border-radius at the left top of the section. Match the text font-weight as the design.

| Your Design | Figma Design |
|-------------|---------------|
| <img width="364" alt="image" src="https://user-images.githubusercontent.com/34578631/171453230-46117cc6-27a4-4632-8ba5-b657752580ac.png"> | <img width="363" alt="image" src="https://user-images.githubusercontent.com/34578631/171453479-070626bc-0fb1-4da6-a444-cd07e3c47277.png"> |